core-services/monitoring/README.md

```diff
@@ -0,0 +1,3 @@
+# Prow Monitoring
+
+This directory will hold all of the manifests for the monitoring and alerting stack we use to monitor Prow.
```

Comment:
@stevekuznetsov Given this is a monitoring for **Prow** specifically, would it make sense for it to live in `core-services/prow`?